### PR TITLE
ACD computed only with NORMAL_CLEARING in variable_last_bridge_hangup…

### PR DESCRIPTION
**Pull request by phildeb**
…_cause

### CODE REVIEW

File: mod_prometheus.rs [hunk 1]
```diff
@@ -316,32 +316,36 @@ fn prometheus_load(mod_int: &ModInterface) -> Status {
...
+                    if parsed_time.is_ok() {
+                        let bill_seconds = parsed_time.unwrap() ;
+                        if let Some(direction) = e.header("Call-Direction") {
+                            if direction == "outbound" {
+                                COUNTERS[FSCounter::SessionsOutboundCallDurationTotal].lock().unwrap().increment_by(bill_seconds as f64);
+                                COUNTERS[FSCounter::SessionsOutboundCallHangupComplete].lock().unwrap().increment();
+                                let acd = COUNTERS[FSCounter::SessionsOutboundCallDurationTotal].lock().unwrap().value() /
+                                                                COUNTERS[FSCounter::SessionsOutboundCallHangupComplete].lock().unwrap().value();
+                                GAUGES[FSGauge::SessionsOutboundACD].lock().unwrap().set(acd);
+                            }else if direction == "inbound"{
```
Comment (grumvalski) [inline mod_prometheus.rs:332]: use uniform code formatting, put a space after each "}" and a space before each "{".

File: mod_prometheus.rs [hunk 2]
```diff
@@ -316,32 +316,36 @@ fn prometheus_load(mod_int: &ModInterface) -> Status {
...
+                                COUNTERS[FSCounter::SessionsInboundCallDurationTotal].lock().unwrap().increment_by(bill_seconds as f64);
+                                COUNTERS[FSCounter::SessionsInboundCallHangupComplete].lock().unwrap().increment();
+                                let acd = COUNTERS[FSCounter::SessionsInboundCallDurationTotal].lock().unwrap().value() /
+                                                COUNTERS[FSCounter::SessionsInboundCallHangupComplete].lock().unwrap().value();
+                                GAUGES[FSGauge::SessionsInboundACD].lock().unwrap().set(acd);
+                            }
+                        }
+                    }else {
```
Comment (grumvalski) [inline mod_prometheus.rs:340]: same as above and for the others.

File: mod_prometheus.rs (hunk 1 above)
Comment (phildeb) [reply mod_prometheus.rs]: done

File: mod_prometheus.rs (hunk 2 above)
Comment (phildeb) [reply mod_prometheus.rs]: done

**Review by grumvalski — APPROVED** [review]

**Pull request merged**